tests/mocha/block_test.js

```diff
@@ -1376,6 +1380,7 @@ suite('Blocks', function() {
         this.block.addIcon(new MockIconA());
         chai.assert.throws(
             () => {this.block.addIcon(new MockIconA());},
+            '',
```

Comment:
I think neither of these is quite correct? https://www.chaijs.com/api/assert/#method_throws

Reply:
I think I'm gonna define a new error class for this. That seems like the correct-est + most useful thing to do.